devel/revive-hf.lua

```diff
@@ -0,0 +1,40 @@
+-- Revives the specified historical figure
+
+--[====[
+
+devel/revive-hf
```

Comment:
is this script intended for developers instead of players? it seems to me that it could just go in the root scripts dir.

Reply:
Yeah, mainly because it doesn't give the hf a new body yet.

Reply:
could we work that functionality it so that the script is generally useful? what more do you think would need to be done to make it worthy of the top dir?